src/audio/volume/volume.c

```diff
@@ -243,6 +243,24 @@ static inline int32_t volume_windows_fade_ramp(struct vol_data *cd, int32_t ramp
 }
 #endif
 
+void volume_set_ramp_channel(struct vol_data *cd, uint32_t channels_count)
```

Comment:
Maybe function name should be instead `volume_set_ramp_channel_counter() `to avoid confusion what it does.